### DISCUSSION TITLE
Disable on/off

### DISCUSSION THREAD

**Discussion by bartplessers** [arendst/Tasmota#15283] [Q&A] [answered]
Hi,

I'm using a sonoff pow r2 flashed with tasmota only for power measurement.

The Home Assistant works perfect, but I want to prevent power on/off of the device. (Yesterday I unfortunately powered off my server... 🙄)

Is there a possibility to disable the switch functionality of the sonoff device in Tasmota?

Kind regards 
Bart Plessers

**Comment by ascillato** [comment, marked as answer]
Yes, please take a look at the docs at https://tasmota.github.io/docs/Commands/#PowerOnState

Please, go to the Tasmota console and type `poweronstate 4` that is **turn power(s) ON and disable further power control**

**Reply by bartplessers** [reply]
Ah, perfect! Thanx!

**Reply by domdfcoding** [reply]
Is there away to hide the (now non-functional) "Toggle" button and "ON" text in the web UI with `poweronstate 4`? Clicking "Toggle" correctly does nothing but it's then confusing having it still show up.

**Reply by sfromis** [reply]
Straight answer, no, the web UI is not meant as a dashboard to be tweaked for daily operation.

If you want to stop seeing it as a Power output, you could instead config the gpio as `Output Hi`.
